pymongo/collection.py

```diff
@@ -2364,7 +2364,10 @@ def list_search_indexes(
             pipeline = [{"$listSearchIndexes": {"name": name}}]
 
         coll = self.with_options(
-            codec_options=DEFAULT_CODEC_OPTIONS, read_preference=ReadPreference.PRIMARY
+            codec_options=DEFAULT_CODEC_OPTIONS,
+            read_preference=ReadPreference.PRIMARY,
+            write_concern=WriteConcern(),
+            read_concern=ReadConcern(),
```

Comment:
Let's use the DEFAULT_READ_CONCERN/DEFAULT_WRITE_CONCERN objects.

Reply:
Done